documentation/source/user_section/tutorials/4d/processing-fmri-data/preprocessing.rst

```diff
@@ -0,0 +1,83 @@
+Preprocessing steps to highlight the spinal cord
+################################################
+
+Similarly to the dMRI tutorial, prior to motion correction, it is useful to highlight the spinal cord region using a mask, as it helps to improve accuracy and speed up processing.
+
+.. figure:: https://raw.githubusercontent.com/spinalcordtoolbox/doc-figures/jn/2857-add-remaining-tutorials/processing-fmri-data/preprocessing.png
+   :align: center
+
+
+Computing a mean image
+----------------------
+
+First we compute the mean of the 4D fMRI data to obtain a coarse 3D approximation. This step is necessary because the relevant SCT tools are designed for individual 3D volumes, rather than 4D images.
+
+.. code::
+
+   sct_maths -i fmri.nii.gz -mean t -o fmri_mean.nii.gz
+
+:Input arguments:
+   - ``-i`` : The input image.
+   - ``-mean`` : The dimension to compute the mean across. In this case, ``sct_maths`` will assume that the fMRI image is a 4D stack of 3D volumes, with dimension ``[x, y, z, t]``. Therefore, ``-mean t`` will average the 3D volumes across the temporal dimension.
+   - ``-o``: The filename of the output image.
+
+:Output files/folders:
+   - ``fmri_mean.nii.gz`` : An 3D image containing the mean of the individual volumes in the 4D dMRI image.
+
+
+Generating a spinal cord segmentation
+-------------------------------------
+
+Due to the low contrast between spinal cord and cerebrospinal fluid of fMRI data, it is difficult to directly obtain a spinal cord segmentation for fMRI data using ``sct_deepseg_sc``. So, as a workaround, we will instead obtain a spinal cord segmentation for another contrast.
+
+.. code::
+
+   cd ../t2
+   sct_deepseg_sc -i t2.nii.gz -c t2
+
+:Input arguments:
+   - ``-i`` : Input image
+   - ``-c`` : Contrast of the input image
+
+:Output files/folders:
+   - ``t2_seg.nii.gz`` : 3D binary mask of the segmented spinal cord
+
+
+Now that we have the spinal cord segmentation for the T2 image, we can transform that segmentation into the space of the fMRI data by registering the T2 segmentation with the fMRI mean image created earlier.
+
+.. code::
+
+   cd ../fmri
+   sct_register_multimodal -i ../t2/t2_seg.nii.gz -d fmri_mean.nii.gz -identity 1
+
+
+:Input arguments:
+   - ``-i`` : Input image
+   - ``-d`` : Destination image
+   - ``-identity`` : Just put source into destination (no optimization). (TODO: This description is a little vague/confusing for me. How does this mean ``-identity``? If no optimiation is performed, then what does this accomplish? Can we clarify this in the help description of ``sct_register_multimodal``.)
```

Comment:
I was little confused by the help description of the `-identity` parameter... I wasn't sure how to explain _why_ we use this here for this step. :sweat_smile:

Reply:
Thank you for the clarification! I've tried rewriting what you've said to make sure I fully understand:

`   - ``-identity`` : Supplying this option will skip cord shape optimizations (e.g. translations, rotations, deformations) during registration. Instead, the registration will only change the properties of the input image (dimension, resolution, orientation) so that they match the properties of the destination image. (Conceptually, this is like copying the source data into the destination data without change, hence the name 'identity'.)`

Does that sound about right?

Reply:
from my understanding yes! maybe @jcohenadad can validate!

Reply:
how about:
- ``-identity``: Supplying this option will skip registration optimization (e.g. translations, rotations, deformations) and will only rely on the qform (from the NIfTI header) of the source and destination images. Use this option if you wish to put the source image into the space of the destination image (i.e. match dimension, resolution and orientation).`